ReleaseTooling/Sources/FirebaseManifest/FirebaseManifest.swift

```diff
@@ -25,7 +25,11 @@ public let shared = Manifest(
   pods: [
     Pod("FirebaseSharedSwift"),
     Pod("FirebaseCoreDiagnostics", zip: true),
+    Pod("FirebaseCoreInternal", zip: true),
```

Comment:
These are only headers, right? Do they need to be in the zip?

Reply:
Good point. We need the dependencies, but not the binaries. The change in CocoaPodUtils.swift is probably closer.